account/update_account_with_account_key_public/boilerplate.js

```diff
@@ -0,0 +1,110 @@
+let recipientAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+
+/**
+ * Boilerplate code about "How to Update Klaytn Account Keys with Caver #1 — AccountKeyPublic"
+ * Related article - Korean: https://medium.com/klaytn/caver-caver%EB%A1%9C-klaytn-%EA%B3%84%EC%A0%95%EC%9D%98-%ED%82%A4%EB%A5%BC-%EB%B0%94%EA%BE%B8%EB%8A%94-%EB%B0%A9%EB%B2%95-1-accountkeypublic-7f8a7197e2d4
+ * Related article - English: https://medium.com/klaytn/caver-how-to-update-klaytn-account-keys-with-caver-1-accountkeypublic-30336b8f0b50
+ */
+async function main() {
+    try {
+        loadEnv()
+        await run()
+    } catch (err) {
+        console.error(err)
+    }
+}
+
+main()
+
+function loadEnv() {
+    envs = dotenv.config({ path: `${ROOT_DIR}/.env` })
+    if (envs.error) {
+        throw envs.error
+    }
+
+    nodeApiUrl = nodeApiUrl === '' ? envs.parsed.NODE_API_URL : nodeApiUrl
+    accessKeyId = accessKeyId === '' ? envs.parsed.ACCESS_KEY_ID : accessKeyId
+    secretAccessKey = secretAccessKey === '' ? envs.parsed.SECRET_ACCESS_KEY : secretAccessKey
+    chainId = chainId === '' ? envs.parsed.CHAIN_ID : chainId
+    senderAddress = senderAddress === '' ? envs.parsed.SENDER_ADDRESS : senderAddress
+    senderPrivateKey = senderPrivateKey === '' ? envs.parsed.SENDER_PRIVATE_KEY : senderPrivateKey
+    recipientAddress = recipientAddress === "" ? envs.parsed.RECIPIENT_ADDRESS : recipientAddress;
+}
+
+async function run() {
+    console.log(`=====> Update AccountKey to AccountKeyPublic`)
+    const option = {
+        headers: [
+            {
+                name: 'Authorization',
+                value: 'Basic ' + Buffer.from(accessKeyId + ':' + secretAccessKey).toString('base64')
+            },
+            {name: 'x-chain-id', value: chainId},
+        ]
+    }
+    const caver = new Caver(new Caver.providers.HttpProvider(nodeApiUrl, option))
+
+    // Add keyring to in-memory wallet
+    const senderKeyring = caver.wallet.keyring.create(senderAddress, senderPrivateKey)
+    caver.wallet.add(senderKeyring)
+
+    // Create new private key
+    const newKey = caver.wallet.keyring.generateSingleKey()
+    console.log(`new private key: ${newKey}`)
+
+    // Create new Keyring instance with new private key
+    const newKeyring = caver.wallet.keyring.create(senderKeyring.address, newKey)
+    // Create an Account instance that includes the address and the public key
+    const account = newKeyring.toAccount()
+    console.log(account)
+
+    // Create account update transaction object
+    const accountUpdate = caver.transaction.accountUpdate.create({
+        from: senderKeyring.address,
+        account: account,
+        gas: 50000,
+    })
+
+    // Sign the transaction
+    await caver.wallet.sign(senderKeyring.address, accountUpdate)
+    // Send transaction
+    const receipt = await caver.rpc.klay.sendRawTransaction(accountUpdate)
+    console.log(`Account Update Transaction receipt => `)
+    console.log(receipt)
```

Comment:
How about adding `console.log` to let users know about they need to replace `SENDER_PRIVATE_KEY` in .env file?

Reply:
@jimni1222 
That will be great.
How about adding console.log like this? 
"You must replace `SENDER_PRIVATE_KEY` in `caver-js-exapmles/.env` with newly-updated private key to use the updated account continuously.".